app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
 
+    people = {}
```

Comment:
Using a class-level dictionary `people` to store `Person` instances by name assumes that names are unique. If two people have the same name, the latter will overwrite the former in the dictionary. Consider using a unique identifier for each person.